src/WebJobs.Script/BindingExtensionBundle/ExtensionBundleOptionsSetup.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Azure.WebJobs.Script.Configuration;
+using Microsoft.Azure.WebJobs.Script.Properties;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using NuGet.Packaging;
+using NuGet.Versioning;
+
+namespace Microsoft.Azure.WebJobs.Script.BindingExtensionBundle
+{
+    internal class ExtensionBundleOptionsSetup : IConfigureOptions<ExtensionBundleOptions>
+    {
+        private readonly IConfiguration _configuration;
+
+        public ExtensionBundleOptionsSetup(IConfiguration configuration)
+        {
+            _configuration = configuration;
+        }
+
+        public void Configure(ExtensionBundleOptions options)
+        {
+            IConfigurationSection jobHostSection = _configuration.GetSection(ConfigurationSectionNames.JobHost);
+            var extensionBundleSection = jobHostSection.GetSection(ConfigurationSectionNames.ExtensionBundle);
+
+            // value of extension bundle property is set to null or a string
+            if (extensionBundleSection.Value != null)
```

Comment:
This check is somewhat confusing - is it because the section is supposed to be an object, but if accidentally set to a string or null you want to validate that?

Reply:
Can you use [ConfigurationExtensions.Exists](https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.configuration.configurationextensions.exists?view=aspnetcore-2.0#Microsoft_Extensions_Configuration_ConfigurationExtensions_Exists_Microsoft_Extensions_Configuration_IConfigurationSection_) to check whether bundle section is defined or not?

Reply:
@mathewc , Yes you are correct. 
@pragnagopa I cannot use0` ConfigurationExtensions.Exists`. Because it does not differentiate between the property having set to null/string and it having children. However in our case extensionBundleSection can have children but cannot hold a direct value.

Reply:
In that case we would still have to check `extensionBundleSection.Value != null` because it cannot have a direct value.